kafka-ui-e2e-checks/pom.xml

```diff
@@ -267,19 +267,10 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
-                        <version>${maven.surefire-plugin.version}</version>
                         <configuration>
                             <skipTests>true</skipTests>
                         </configuration>
                     </plugin>
-                    <plugin>
```

Comment:
where did this plugin go? for local profile

Reply:
The Maven compiler plugin is available by default and the version and the Java release are defined in the parent POM of the `kafka-ui-e2e-checks` project:


https://github.com/provectus/kafka-ui/blob/1f65a905b8e23d493114681c7b5772d6a8a65d51/pom.xml#L14

https://github.com/provectus/kafka-ui/blob/1f65a905b8e23d493114681c7b5772d6a8a65d51/pom.xml#L92-L96

This being said, happy to add the plugin back for clarity.

Reply:
Nah, that's alright, thank you